src/lucky_record/charms/json_extensions.cr

```diff
@@ -3,6 +3,8 @@ struct JSON::Any
     alias ColumnType = JSON::Any
     include LuckyRecord::Type
 
+    alias JSONType = Array(JSONType) | Bool | Float64 | Hash(String, JSONType) | Int64 | String | Nil | Int32 | Float32
```

Comment:
I *think* you can add `NamedTuple(Symbol, JSONType)` if you use `to_json` to do the conversions